src/aims_ui/page_typeahead.py

```diff
@@ -37,6 +37,9 @@ def typeahead():
       all_user_input.get(page_name),
   )
 
+  if result == 'error_connecting':
```

Comment:
Likewise, this looks like more error-checking code that could just be a `catch`.